plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Drilldown.js

```diff
@@ -40,9 +40,16 @@ Ext.define('NX.controller.Drilldown', {
 
   permission: undefined,
 
+  /**
+   * @override
```

Comment:
override here seems odd, I'd expect to see sub-classes use this anno, but not here on the def.  Here public/protected probably more useful.  Unsless I'm missing that the super-class has a getDescription which this class is overriding.